Arduino/startUpwithBluetooth/startUpwithBluetooth.ino

```diff
@@ -0,0 +1,70 @@
+#include <Smartcar.h>
+
+const unsigned short LEFT_ODOMETER_PIN = 2;
+const unsigned short RIGHT_ODOMETER_PIN = 3;
+const unsigned long PRINT_INTERVAL = 100;
+unsigned long previousPrintout = 0;
+const int TRIGGER_PIN = 6; //D6
+const int ECHO_PIN = 7; //D7
+const unsigned int MAX_DISTANCE = 100;
+char blue;
+SR04 front(TRIGGER_PIN, ECHO_PIN, MAX_DISTANCE);                //bluetooth value reciever
+
+
+BrushedMotor leftMotor(8, 10, 9);
+BrushedMotor rightMotor(12, 13, 11);
+DifferentialControl control(leftMotor, rightMotor);
+
+GY50 gyroscope(37);
```

Comment:
`37` is probably not your correct offset. Please find the appropriate value for your sensor.

Reply:
I have calculated the values for our car and the offset for our car for the gyroscope was 18.